R/utils.R

```diff
@@ -424,7 +424,7 @@ verify_attr_names <- function(p) {
   # some layout attributes (e.g., [x-y]axis can have trailing numbers)
   attrs_name_check(
     sub("[0-9]+$", "", names(p$x$layout)),
-    c(names(Schema$layout$layoutAttributes), c("barmode", "bargap", "bargroupgap", "mapType")),
+    c(names(Schema$layout$layoutAttributes), c("barmode", "bargap", "bargroupgap", "barcornerradius", "mapType")),
```

Comment:
Just curious how did you know to change this?

Reply:
I changed it for bargroup gap a while ago as well. If you try to use barcornerradius you get an ugly warning. I set option(warning=2) to make it an error and checked the stack trace

Reply:
Nice. So that means we could get rounded corners just from this PR?

Reply:
Yep, rounded corners will work with this. Is there anything else we needed?

Reply:
Would you be able to test out the “between” option for the “layer” setting: https://www.notion.so/displayr/Our-plotly-version-is-old-which-means-that-some-features-are-unavailable-b20b3ebe92354cc583d599f1a78768af?pvs=4#c4bb2c738cd5406787016e71fccdbeae